docs/src/components-dark.dot

```diff
@@ -1,38 +1,46 @@
-digraph {
-  graph [bgcolor=transparant, fontname="SFMono-Regular"]
+ digraph {
```

Comment:
Was this extra space at the start intentional? The `components-light.dot` does not have it! (this is about the only thing I can say about this PR, I guess the generated image is fine ;))

Reply:
Not intentional! Thanks!